XamarinCommunityToolkit/Views/CameraView/Android/ImageAvailableListener.android.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Android.Media;
+using AImage = Android.Media.Image;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	class ImageAvailableListener : Java.Lang.Object, ImageReader.IOnImageAvailableListener
+	{
+		public event EventHandler<byte[]> Photo;
+
+		public void OnImageAvailable(ImageReader reader)
+		{
+			AImage image = null;
+
+			try
+			{
+				image = reader.AcquireNextImage();
+				var buffer = image.GetPlanes()[0].Buffer;
+				var imageData = new byte[buffer.Capacity()];
+				buffer.Get(imageData);
+
+				Photo?.Invoke(this, imageData);
+				buffer.Clear();
+			}
+			catch (Exception)
+			{
```

Comment:
I've noticed a couple of catch blocks that just swallow the exception.  Should we be doing this?

Reply:
I think this is to prevent the control to crash the application. But I have no idea